Mobile Test Runner for Jama/Mobile Test Runner for Jama/UserModel.swift

```diff
@@ -14,6 +14,19 @@ class UserModel {
     var firstName: String = ""
     var lastName: String = ""
     var email: String = ""
-    var id: Int = 0
+    var id: Int = -1
     var username: String = ""
+
+    func extractUser(fromData: [String : AnyObject]) {
+        if fromData["firstName"] == nil || fromData["lastName"] == nil || fromData["email"] == nil || fromData["username"] == nil || fromData["id"] == nil {
```

Comment:
This may be too strict of a test. some of these fields may not be required. I need to look into this.